.eslintignore

```diff
@@ -1,11 +1,2 @@
-dist/**
-test/**
-src/plugins/*/static/**
-# Following files will be removed from this list in https://educandu.atlassian.net/browse/EDU-936
-src/index.js
-src/test-helper.js
-src/data/language-name-provider.js
-src/bootstrap/server-bootstrapper.js
-src/components/history-control-panel.js
-src/components/localization/language-select.js
-src/components/localization/language-flag-and-name.js
+/dist/**
+/test-app/dist/**
```

Comment:
I recently learnt that in `.gitignore` and similar files the slash at the beginning means: starting from the directory the ignore file is in. Without the slash it matches any part of the path. So I'd propose to be more specific in the future like above.